vars.sample

```diff
@@ -50,10 +50,20 @@ ORACLE_TABLESPACES_DIR=/home/oracle/app/oracle/oradata
 # }}}
 
 # Only needed for ETL
+ETL_SCRIPTS=/path/to/transmart-data/samples/postgres
+export PATH=$ETL_SCRIPTS:$PATH
```

Comment:
It still doesn't change anything. Unless you're copying the scripts around, you already know exactly where they are. See http://stackoverflow.com/questions/59895/can-a-bash-script-tell-what-directory-its-stored-in